src/trader.js

```diff
@@ -945,7 +917,30 @@ async function UpdateOpenTrades() {
     })
 }
 
+async function UpdateMarginPairs() {
+    return new Promise((resolve, reject) => {
+        axios
+            .get(
+                "https://www.binance.com/gateway-api/v1/friendly/margin/symbols"
+            )
+            .then((res) => {
+                let btconly = res.data.data.filter(
```

Comment:
The BVA strategy only uses BTC quoted pairs, yes, but that isn't a requirement for strategies, is it @jsappme?
So I propose we should remove this filter.

Reply:
yes feel free to remove this filter